### PR TITLE
Issue #51: said_uno

### PR DESCRIPTION
**Pull request by AurumTheEnd**

### CODE REVIEW

**Comment by AurumTheEnd** [comment]
`fn test_active_cards()` extremely rarely randomly fails, see if you can spot a mistake in it

**Comment by holubond** [comment]
> `fn test_active_cards()` extremely rarely randomly fails, see if you can spot a mistake in it

Are you testing me, or genuinely don't know? :D

**Comment by AurumTheEnd** [comment]
> Are you testing me, or genuinely don't know? :D
It was too late for me to grasp where the problem may lie, hoped you could genuinely clear it up :D

**Comment by AurumTheEnd** [comment]
Fixed in bc0e904, took me an hour to find.

**Comment by AurumTheEnd** [comment]
@mazanec202 reviewable, mergable

File: endless_test.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+while :
+do
+	if ! cargo test ; then
+	  break
+	fi
+done
```
Comment (holubond) [inline endless_test.sh:11]: Toto je hodně cool :D

**Review by holubond — APPROVED** [review]

File: endless_test.sh (hunk 1 above)
Comment (AurumTheEnd) [reply endless_test.sh:11]: Is it? :D

**Pull request merged**